### PR TITLE
Normalize email first when creating user.

### PR DESCRIPTION
**Pull request by johngian**

### CODE REVIEW

**Comment by johngian** [comment]
@comzeradd r?

File: marketpulse/auth/backend.py [hunk 1]
```diff
@@ -50,6 +50,11 @@ def __init__(self, *args, **kwargs):
 
         self.mozillians_client = MozilliansClientLegacy(self.api_url, self.api_key, self.app_name)
 
+    def _normalize_email(self, email):
+        email_name, domain = email.strip().rsplit('@', 1)
+        email = '@'.join([email_name, domain.lower()])
```
Comment (comzeradd) [inline marketpulse/auth/backend.py:60]: Does it make sense to lower `email_name` too? I think it's case insensitive.

**Comment by comzeradd** [comment]
r+wc

File: marketpulse/auth/backend.py (hunk 1 above)
Comment (johngian) [inline marketpulse/auth/backend.py:60]: RFC for email says that `email_name` is case sensitive. This is also the way django internally normalizes emails.

**Pull request merged**